rllib/policy/policy.py

```diff
@@ -92,6 +94,9 @@ def compute_actions(self,
             explore (bool): Whether to pick an exploitation or exploration
                 action (default: None -> use self.config["explore"]).
             timestep (int): The current (sampling) time step.
+            trajectories (List[Trajectory]): A List of Trajectory data used
+                to create a view for the Model forward call. Only used so far
+                iff `_fast_sampling=True` (only supported for torch).
```

Comment:
This doesn't seem right, we should be able to batch across trajectories right? So this should be a single `TrajectoryView` object that internally batches across both time and multiple trajectories.